### PR TITLE
documentation for http

### LINKED ISSUE #3: webpage does not work
The webpage [https://ssisbit.github.io/ssis.bit/](https://ssisbit.github.io/ssis.bit/) only shows the template, but no information about this project.

### PR DESCRIPTION
**Pull request by kreier**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# ssis.bit
+AIO solution for SSIS students
+
+This is just a placeholder. You might be looking for these files:
+
+[index.md](index.md) The file for the website [ssisbit.github.io/ssis.bit](https://ssisbit.github.io/ssis.bit)
+
+[history.md](history.md) A little history of this project.
+
```
Comment (ssis-robotics) [inline docs/README.md:9]: This looks decent

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+The planned adapterplate adds a 3-way button for input, place to hold the little 350 mAh battery and interface for I2C with regular 2.54mm 4pin JST XH as well as the small Stemma QT 1mm JST SH connector (QWIIC).
+
+## Software
+
+<img src="circuitpython.png" align="right">
+
+The ssis:bit runs [CircuitPython](https://circuitpython.org/) and can be easily programmed with the [Mu Editor](https://codewith.mu/en/). There is a lot to learn on [Adafruit with CircuitPython]() and with their great libraries there is less frustration getting things started.
+
+## History
+
```
Comment (ssis-robotics) [inline docs/index.md:19]: Please update with an actual photo of your prototype

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+More will be updated in the [docs/history](docs/history.md) document, but the idea is from 2020 with the [t-display](https://github.com/kreier/t-display) board. Similar idea: Not just the Microcomputer but included power supply (LiPo), display (just 240x135 color) and input device (not keyboard, but 3 buttons) to have it always with you and ready to use.
```
Comment (ssis-robotics) [inline docs/index.md:20]: This document does not exist yet.

File: docs/README.md (hunk 1 above)
Comment (ssisbit) [inline docs/README.md:9]: Thanks for inlcuding the /docs structure. But be aware: at least /README.md and /docs/index.md have to be updated regularly.

What about a webpage https://ssisbit.org/ like for the micro:bit ?

File: docs/index.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# ssis.bit
+AIO solution for SSIS students
+
+## Hardware
+
+CircuitPython supports [more than 200 different boards](https://circuitpython.org/downloads). Locally available is the [TTGO T8 ESP32-S2 ST7789](https://circuitpython.org/board/lilygo_ttgo_t8_s2_st7789/) at several online stores. The board includes the powerful 240MHz CPU, 4MB Flash and 8MB PSRAM, a 1.14" ST7789 display, a microSD card slot for virtually unlimited local data storage, battery connector 1.25mm JST with charge controller and regular USB-C connector. And buildin WiFi, of course.
+
+![LILYGO T8](T8.jpg)
```
Comment (ssisbit) [inline docs/index.md:8]: Hope this works

File: docs/index.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+The planned adapterplate adds a 3-way button for input, place to hold the little 350 mAh battery and interface for I2C with regular 2.54mm 4pin JST XH as well as the small Stemma QT 1mm JST SH connector (QWIIC).
+
+## Software
+
+<img src="circuitpython.png" align="right">
+
+The ssis:bit runs [CircuitPython](https://circuitpython.org/) and can be easily programmed with the [Mu Editor](https://codewith.mu/en/). There is a lot to learn on [Adafruit with CircuitPython]() and with their great libraries there is less frustration getting things started.
```
Comment (ssisbit) [inline docs/index.md:16]: Make some subfolders like for /basics /games /sensors and the like

File: docs/index.md (hunk 3 above)
Comment (ssisbit) [inline docs/index.md:20]: Let's get started!

**Pull request merged**